traffic/algorithms/navigation.py

```diff
@@ -209,6 +209,59 @@ def on_runway(self, airport: Union[str, "Airport"]) -> Optional["Flight"]:
             default=None,
         )
 
+    @flight_iterator
+    def takeoff(
+        self, airport: Union[str, "Airport"]
+    ) -> Iterator["Flight"]:
+        """Iterates on all segments of trajectory matching a zone around a runway of the
+        given airport.
+        Example usage:
+        >>> 
+        """
+        
+        from shapely.geometry import Polygon
```

Comment:
These two imports should be moved to the top of the file.
`traffic.core.geodesy` should be `..core.geodesy`